### DISCUSSION TITLE
How can i prevent to change background after login?

### DISCUSSION THREAD

**Discussion by root9191** [JaKooLit/Hyprland-Dots#320] [Q&A]
Hi, use hyprlock and that uses normally the current wallpaper, but on first startup it uses the "last" wallpaper and right after login i see the new one. Can i somehow fix this?

**Comment by JaKooLit** [comment]
disable the automatic wallpaper change


https://github.com/JaKooLit/Hyprland-Dots/wiki/FAQ#%EF%B8%8F-automatic-wallpaper-change
